docs/providers.rst

```diff
@@ -0,0 +1,111 @@
+Providers for GCP-related Services
+==================================
+
+.. TODO: externally link "interfaces" to gordon-dns's docs once published.
+
+This package provides the following interface providers. Use any or all of them; they are not dependent on each other.
+
+
+Provider APIs
+-------------
+
+.. toctree::
+   :maxdepth: 1
+
+   compute
+   dns
+   pubsub
+
+
+Example Gordon Configuration with GCP Plugin
+--------------------------------------------
+
+.. literalinclude:: ../gordon.toml.example
+   :language: ini
```

Comment:
The `literalinclude` sphinx directive includes the file itself and highlights it according to `language` defined (from pygments).